lang/en.json

```diff
@@ -870,6 +871,7 @@
         "faceted": {
             "selected": {
                 "title": "Refine by",
+                "facet-label": "Location of applied filters",
```

Comment:
@bc-vlad-dlogush up to you, but my suggestion is just `"Applied filters"`

Reply:
@VladDlogush I agree with Yevhenii, please update region name to "Applied filters"

Reply:
done!